### PR TITLE
Add odo service list -o json message

### LINKED ISSUE #3241: Namespace name missing in odo service list -o json message
/kind bug

## What versions of software are you using?

**Operating System:**
All supported

**Output of `odo version`:**
master

## How did you run odo exactly?

```
$ odo service list
odo service list            
 ✗  No operator backed services found in the namesapce

odo service list -o json
{
    "kind": "Error",
    "apiVersion": "odo.dev/v1alpha1",
    "metadata": {
        "creationTimestamp": null
    },
    "message": "No operator backed services found in the namesapce"
}
```

## Actual behavior

$ odo service list
odo service list            
 ✗  No operator backed services found in the namesapce

odo service list -o json
{
    "kind": "Error",
    "apiVersion": "odo.dev/v1alpha1",
    "metadata": {
        "creationTimestamp": null
    },
    "message": "No operator backed services found in the namesapce"
}

## Expected behavior

$ odo service list
odo service list            
 ✗  No operator backed services found in the namesapce **<namespace_Name>**

odo service list -o json
{
    "kind": "Error",
    "apiVersion": "odo.dev/v1alpha1",
    "metadata": {
        "creationTimestamp": null
    },
    "message": "No operator backed services found in the namesapce **<namespace_Name>**"
}

### PR DESCRIPTION
**Pull request by jichenjc**
**What type of PR is this?**

/kind bug
**What does does this PR do / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #3241 

**How to test changes / Special notes to the reviewer**:
odo service list -o json

### CODE REVIEW

File: tests/integration/operatorhub/cmd_service_test.go [hunk 1]
```diff
@@ -263,8 +263,11 @@ spec:
 			// Now let's check the output again to ensure expected behaviour
 			stdOut = helper.CmdShouldFail("odo", "service", "list")
 			jsonOut = helper.CmdShouldFail("odo", "service", "list", "-o", "json")
-			Expect(stdOut).To(ContainSubstring("No operator backed services found in the namesapce"))
-			helper.MatchAllInOutput(jsonOut, []string{"No operator backed services found in the namesapce", "\"message\": \"No operator backed services found in the namesapce\""})
+
+			msg := fmt.Sprintf("No operator backed services found in namesapce: %s", project)
+			msgwithquota := fmt.Sprintf("\"message\": \"No operator backed services found in namespace: %s\"", project)
```
Comment (dharmit) [inline tests/integration/operatorhub/cmd_service_test.go:268]: You meant `msgWithQuote`?

Comment (jichenjc) [reply tests/integration/operatorhub/cmd_service_test.go:268]: yea... thanks for pointing out..

**Comment by dharmit** [comment]
@jichenjc thanks for making the change. Can you please rebase your PR to the latest master? I'm asking this just to be sure that changes from #3459 are pulled in.

**Comment by jichenjc** [comment]
@dharmit thanks for the reminder, I updated the PR by rebase master just now

**Pull request merged**